tests/lib/linter/linter.js

```diff
@@ -3408,6 +3425,52 @@ var a = "test2";
             });
         });
 
+        describe("physicalFilenames", () => {
+            it("should allow physicalFilename to be passed on options object", () => {
+                const physicalFilenameChecker = sinon.spy(context => {
+                    assert.strictEqual(context.getPhysicalFilename(), "foo.js");
+                    return {};
+                });
+
+                linter.defineRule("checker", physicalFilenameChecker);
+                linter.verify("foo;", { rules: { checker: "error" } }, { physicalFilename: "foo.js" });
+                assert(physicalFilenameChecker.calledOnce);
+            });
```

Comment:
Do we actually want to allow this? In my understanding, this shouldn't be a new option in the Linter API, and the tests should pass `filename` to `linter.verify()`.

Reply:
That’s correct. It should not be an option to linter.verify().

Reply:
Updated 👍🏻